lib/main.dart

```diff
@@ -11,7 +12,131 @@ class RationCalculatorApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: CowCharacteristicsPage(),
+      theme: ThemeData(
+        useMaterial3: true,
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.green),
+      ),
+      home: const HomeScreen(),
+    );
+  }
+}
+
+class Destination {
```

Comment:
Maybe move these classes out of main, and rather import them.

Reply:
done!